Clean up variables and comments

NBumgardner opened the pull request:
This small PR **improves readability of code and comments** in the first main page *MissionState.hx* and **fixes comments** in the *menus* folder.

### MissionState.hx Changes
Includes 1 new variable, **deadzoneWidth = tileSize * deadzoneBorderTiles**.

* Adds temporarily local variable *deadzoneWidth* into *initCamera()* to improve readability.
* Replaces 64 with existing constant *tileSize* in *initMap()* to remove a hardcoded value.
* Fixes grammar in comment above *addAllFlxObjects()*.

### menus Folder Changes
Fixes comments above functions:

* Typos
* Missing punctuation
* Irregular indentation (if a sentence continues over >1 line, it has 1 more space of indentation)

### Notes
Since there is not a visible **develop** branch, this PR is for the **master** branch.

NBumgardner committed:
Add/Use vars in MissionState

Changes are equivalent:
* Replace hardcoded `64` with global var `tileSize` for initMap().
* Add local var `deadzoneWidth` for initCamera() borders.
* Fix if-else indentation.

Tested. Builds & runs on: Neko, html5.
Flash builds, but when I run it, it opens a browser and
 downloads a `.swf` file I do not know how to use.

NBumgardner committed:
Fix typos in comments, x2

NBumgardner committed:
Fix typos & indentation in menus folder comments

SamBumgardner commented:
I think your PR's good to go now!

If you want to do some extra stuff, there are a few typos in the README that could use fixing:
* I misspelled overwhelming as "overwhelmign" in the line after the "Overview" header.
* The line after the "Getting Started" header says Potion-Shop instead of Strategy-Game.

If you don't feel like it, let me know and I'll merge this in as-is and take care of that stuff later.

Thanks Noah!

NBumgardner committed:
Merge branch 'master' of https://github.com/NBumgardner/Strategy-Game into clean-up-variables-and-comments

NBumgardner committed:
Fix 2 typos in README.md

Fix seen by SamBumgardner during PR#8, 3/5/2017.

NBumgardner committed:
Change branch name from `develop` to `master`

Matches the main, stable branch name used in this project so far.

There has not been a public `develop` branch yet,
 so `master` is more appropriate.
Could also have been changed to: "`master` or `develop`".

NBumgardner commented:
You're welcome Sam! Your 2 fixes have been added, plus 1 more update to the readme.

#### Summary of Last 3 Commits
1. Include PR#9 changes (no conflicts)
2. Fixed 2 Readme typos: `overwhelmign`, `Potion-Shop`
3. Update the readme **Contribution guidelines**: `develop` --> `branch`

I swapped `develop` with `master` because only the latter is a public branch name.

#### Ready to Merge
The project still builds and runs for the *neko* and *html5* platforms.

It is ready to be merged.

SamBumgardner commented:
Thanks, I'll merge it in!
